actions_std.go

```diff
@@ -2632,10 +2632,7 @@ func scriptingActions() {
 		make: func(args []actionArgument) []plistData {
 			var urlItems []plistData
 			for _, item := range args {
-				urlItems = append(urlItems, plistData{
-					dataType: Text,
-					value:    item.value,
-				})
+				urlItems = append(urlItems, attachmentValues("", item.value.(string), Text),)
```

Comment:
Just a redundant comma at the end of the arguments for `attachmentValues()`

Reply:
Also, I think using this here might be a bit cleaner so we don't need to coerse the value if we don't have to.

```suggestion
				urlItems = append(urlItems, paramValue("", arg, String, Text))
```